unit/util/interval/modulo.cpp

```diff
@@ -52,6 +71,19 @@ SCENARIO("modulo interval domain", "[core][analyses][interval][modulo]")
             constant_interval_exprt(CEV(-10), CEV(20)),
             constant_interval_exprt(CEV(0), CEV(5))) ==
           constant_interval_exprt::top(signedbv_typet(32)));
+
+        REQUIRE(
+          constant_interval_exprt::modulo(
+            constant_interval_exprt(CEV(-10), CEV(20)),
```

Comment:
Do you mean why [-10 20] % [0, 5] = TOP but [-10 20] % [1, 5] = [-10, 20]? I've added a comment that taking modulo on something containing zero gives top since divide by zero